integration_tests/testUtils.ts

```diff
@@ -8,8 +8,7 @@ import TestData from '../server/routes/testutils/testData'
 
 export { resetStubs }
 
-const DEFAULT_ROLES = [`ROLE_${bapvUserRoles.STAFF_USER}`] // TODO sort out ROLE_ prefix
-const DEFAULT_NAME = 'john smith'
```

Comment:
Removing this `DEFAULT_NAME` - it was an addition to what the template implementation had but turns out it's not needed.